src/historywork/WriteVerifiedCheckpointHashesWork.h

```diff
@@ -4,19 +4,20 @@
 
 #pragma once
 
-#include "catchup/CatchupConfiguration.h"
-#include "catchup/VerifyLedgerChainWork.h"
-#include "history/HistoryArchive.h"
-#include "historywork/BatchDownloadWork.h"
-#include "historywork/GetHistoryArchiveStateWork.h"
+#include "ledger/LedgerRange.h"
 #include "util/WriteOnceSharedVar.h"
+#include "work/BatchWork.h"
 #include "work/Work.h"
```

Comment:
I don't think you need "work/Work.h" include?

Reply:
Yep, fixed.